PyQt5-stubs/QtCore.pyi

```diff
@@ -2375,13 +2375,20 @@ class Qt(sip.simplewrapper):
     WindowFullScreen = ...  # type: Qt.WindowState
     WindowActive = ...  # type: Qt.WindowState
 
-    class WindowType(int):
-        def __invert__(self) -> 'Qt.WindowType': ...
-        def __or__ (self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType']) -> 'Qt.WindowFlags': ...
-        def __and__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType']) -> 'Qt.WindowFlags': ...
-        def __xor__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType']) -> 'Qt.WindowFlags': ...
-        def __add__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType']) -> 'Qt.WindowFlags': ...
-        def __sub__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType']) -> 'Qt.WindowFlags': ...
+    class WindowType(object):
+        def __init__(self, f: typing.Union['Qt.WindowType', int]) -> None: ...
+        def __int__(self) -> int: ...
+        def __invert__(self) -> 'Qt.WindowFlags': ...
+        def __add__ (self, other: typing.Union['Qt.WindowType', int]) -> 'Qt.WindowFlags': ...
+        def __sub__ (self, other: typing.Union['Qt.WindowType', int]) -> 'Qt.WindowFlags': ...
+        def __or__ (self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType', int]) -> 'Qt.WindowFlags': ...
+        def __and__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType', int]) -> 'Qt.WindowFlags': ...
+        def __xor__(self, other: typing.Union['Qt.WindowFlags', 'Qt.WindowType', int]) -> 'Qt.WindowFlags': ...
+        def __ror__ (self, other: int) -> 'Qt.WindowFlags': ...
+        def __rand__(self, other: int) -> 'Qt.WindowFlags': ...
+        def __rxor__(self, other: int) -> 'Qt.WindowFlags': ...
+        def __radd__ (self, other: int) -> 'Qt.WindowFlags': ...
+        def __rsub__ (self, other: int) -> 'Qt.WindowFlags': ...
```

Comment:
Why the change in inheritance?
```pycon
>>> issubclass(Qt.WindowType, int)
True
```

Reply:
I thought it would make the code simpler, but actually, everything works fine with int inheritance. I'll revert that part.